mitosheet/src/components/footer/DataSheetTabActions.tsx

```diff
@@ -149,6 +153,14 @@ export default function SheetTabActions(props: {
                 }
             })
         }}/> : undefined,
+        mergeSource ? <DropdownItem key='Edit Merge Configuration' title='Edit Merge Configuration' onClick={() => {
```

Comment:
Why don't we just call this "Edit Merge" or "Update Merge" -- configuration is super unnecessary. Same with Pivot below.